config/regex.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
     TEMPLATE_NAME: /^[\w/-]+$/,
     // Version can only have up to 2 decimals, like 1.2.3
     VERSION: /^(\d+\.)?(\d+\.)?(\d+)$/,
```

Comment:
Should we update the regex for `VERSION` to match the end of the regex for `FULL_TEMPLATE_NAME`? Or at least make them consistent somehow
Like this: `VERSION: /^((\d+)?(\.\d+)?(\.\d+)?)$/,`